src/uct/ib/ud/base/ud_def.h

```diff
@@ -181,8 +184,10 @@ typedef struct uct_ud_ctl_desc {
     ucs_queue_elem_t        queue;       /* Queue element in outstanding queue */
     uint16_t                sn;          /* Sequence number in outstanding queue */
     uct_ud_send_skb_t       *self_skb;   /* Back-pointer to owner skb */
-    uct_ud_send_skb_t       *resent_skb; /* Points to a re-sent skb in the window.
-                                            Can be NULL. */
+    uct_ud_send_skb_t       *resent_skb; /* For resend skb: points to a re-sent
+                                            skb in the window. Can be NULL. */
```

Comment:
maybe do not add trailing `.` as other fields don't have it

Reply:
sure, will unify

Reply:
I thought typical approach is that `.` maybe skipped for a single sentence, but should be used if there are more than one sentences